openhtf/io/frontend/__main__.py

```diff
@@ -244,8 +220,8 @@ def initialize(self, store):
 
   def post(self, host, port, prompt_id):
     msg = json.JSONEncoder().encode(
-      {'id': prompt_id, 'response': self.request.body})
-    self.store.MessageFramework(host, port, method='POST', message=msg)
+        {'id': prompt_id, 'response': self.request.body})
+    self.store[(host, int(port))].Notify(msg)
```

Comment:
no inner parens required: `self.store[host, int(port)].Notify(msg)` should work.